client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,36 @@
+// HelloWorldWidget is an arbitrary name for any "dumb" component. We do not recommend suffixing
```

Comment:
@alexfedoseev I vote we call this thing `HelloWorld` and rename the smart component to `HelloWorldContainer`